web/src/js/components/EventLog.jsx

```diff
@@ -1,31 +1,70 @@
-import React, { PropTypes } from 'react'
+import React, { Component, PropTypes } from 'react'
 import { connect } from 'react-redux'
 import { toggleEventLogFilter, toggleEventLogVisibility } from '../ducks/eventLog'
-import { ToggleButton } from './common'
+import ToggleButton from './common/ToggleButton'
 import EventList from './EventLog/EventList'
 
-EventLog.propTypes = {
-    filters: PropTypes.object.isRequired,
-    events: PropTypes.array.isRequired,
-    onToggleFilter: PropTypes.func.isRequired,
-    onClose: PropTypes.func.isRequired
-}
+class EventLog extends Component {
+
+    static propTypes = {
+        filters: PropTypes.object.isRequired,
+        events: PropTypes.array.isRequired,
+        onToggleFilter: PropTypes.func.isRequired,
+        onClose: PropTypes.func.isRequired,
+        defaultHeight: PropTypes.number,
+    }
+
+    static defaultProps = {
+        defaultHeight: 200,
+    }
+
+    constructor(props, context) {
+        super(props, context)
+
+        this.state = { height: this.props.defaultHeight }
+
+        this.onDragStart = this.onDragStart.bind(this)
```

Comment:
Ok, I saw that you removed the splitter and added it here instead. What's the motivation for this? I think we want to keep that separate and maybe make the Splitter nicer at a later point in time.